src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:44599/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:44599/project/issues?id=sample&pullRequest=6154&issues=AWetP9q1XZYAdQgy0Q21&open=AWetP9q1XZYAdQgy0Q21)

[ISSUE_KEY]: AWetP9q1XZYAdQgy0Q21